package.json

```diff
@@ -16,27 +16,38 @@
   },
   "dependencies": {
     "@expo/vector-icons": "^14.0.2",
+    "@react-native-async-storage/async-storage": "1.23.1",
```

Comment:
async-storage is good, but `react-native-mmkv` is much more performant (see [benchmarks](https://github.com/mrousavy/react-native-mmkv?tab=readme-ov-file#benchmark)). please move to that package instead